online-inference/stable-diffusion/03-inference-service.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: serving.kubeflow.org/v1beta1
+kind: InferenceService
+metadata:
+  name: stable-diffusion
+spec:
+  predictor:
+    containerConcurrency: 1
+    minReplicas: 1
+    maxReplicas: 1 
+    affinity:
+      nodeAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+          - matchExpressions:
+            - key: gpu.nvidia.com/class
+              operator: In
+              values:
+              - RTX_A4000
+            - key: topology.kubernetes.io/region
+              operator: In
+              values:
+              - ORD1 
+    containers:
+      - name: kserve-container
+        image: tweldoncw/stable-diffusion:4
+        command:
+          - "python3"
+          - "/app/service.py"
+          - "--model-id=CompVis/stable-diffusion-v1-4"
+          - "--hf-home=/mnt/models/hub"
+          # The following arguments are defaults which can be changed here and per request
+          - "--precision=float16" # PyTorch dtype/model precision(one of float16 or float32)
+          - "--guidance-scale=7.0" # How well image generation adheres to the prompt. With very large numbers the images might look good, but will be less diverse. 
+          - "--num-inference-steps=50" # Results are better the more steps you use but will take longer.
+          - "--seed=42" # Every time you use the same seed you'll have the same image result. Use random numbers for different results with the same prompt.
```

Comment:
Why are we setting a fixed seed here?

Reply:
All CLI args are here are to show what options are available. However this makes me realize, there is no way to use the default random seed. Will need to change the default for that in `service.py`.

Reply:
Now defaults to random seed with optional deterministic manual seed.